common/static/common/js/spec/discussion/view/response_comment_view_spec.js

```diff
@@ -164,10 +166,17 @@
                             }
                         };
                     });
+
+                    this.event = DiscussionSpecHelper.makeEventSpy();
+                    // All the way down in discussion/utils.js there's this line
+                    // element.after(...);
+                    // element is event.target in this case. This causes a JS exception, so we override the target
+                    this.event.target = $("body");
+                    $(this.event.target).prop("disabled", false);
```

Comment:
This is a bit messy. It would be nicer if you modify `makeEventSpy` so that after creating the `event` mock, it assigns `event.target.after = createSpy('after')` before returning `event`.